decidim-core/vendor/assets/javascripts/datepicker-locales/foundation-datepicker.ar.js

```diff
@@ -1,16 +0,0 @@
-/**
- * Arabic translation for foundation-datepicker
- * Nico Mollet <nico.mollet@gmail.com>
- */
-;(function($){
-	$.fn.fdatepicker.dates['ar'] = {
```

Comment:
@ferblape I see all these files have been removed, where are they now? I can't see them being moved... Thanks

Reply:
Ah, I see `foundation-datepicker` is added as a depndency in the `package.json`